README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
+I am a rising third-year in the College, double majoring in **political science** and **economics** *with a specialization in business economics*. I will also hopefully minor in *near eastern languages and civilizations*.
+
+
+## Backround
+I am orginally from Fairfield, Connecitcut, but went to high school in Riverdale New York at SAR High School.
+
+
+## General Hobbies, interests, and likes/dislikes
+1. Eating all things pumpkin - fall is my favorite time of the year. **pumpkin ice cream**, **pumpkin spice lattes**, **pumpkin pie** the options are endless
```

Comment:
You don't actually have to manually number lists. GitHub will automatically renumber the items: 

```r
1. Eating all things pumpkin
1. Sailing
1. Hiking
```

1. Eating all things pumpkin
1. Sailing
1. Hiking